README.md

```diff
@@ -133,6 +136,7 @@ time_zone: null
 | date_pattern          | string           | **Optional** | Pattern to use for date formatting. If not provided, falls back to a localized default date formatting. See [luxon](https://moment.github.io/luxon/#/formatting?id=table-of-tokens) for valid tokens                              | `D`       |
 | hide_today_section    | boolean          | **Optional** | Hides the cards today section (upper section), containing the large weather icon, clock and current date                                                                                                                          | `false`   |
 | hide_forecast_section | boolean          | **Optional** | Hides the cards forecast section (lower section),containing the weather forecast                                                                                                                                                  | `false`   |
+| hide_humidity             | boolean          | **Optional** | Hides the humidity from the today section                                                                                                                                                                                             | `false`   |
```

Comment:
Please set the default value to `true`. i don't think it makes sense to enable humidity by default for every user